ccpp/config/ccpp_prebuild_config.py

```diff
@@ -231,9 +232,14 @@
     'ccpp/physics/physics/sfc_nst.f'                        : ['physics'],
     'ccpp/physics/physics/sfc_ocean.F'                      : ['physics'],
     'ccpp/physics/physics/sfc_sice.f'                       : ['physics'],
+    'ccpp/physics/physics/mp_fer_hires.F90'                 : ['physics'],
     'ccpp/physics/physics/gmtb_scm_sfc_flux_spec.F90'       : ['physics'],
     }
 
+# Default build dir, relative to current working directory,
+# if not specified as command-line argument
+DEFAULT_BUILD_DIR = 'scm/bin'
```

Comment:
Thank you for making all these changes by looking at my poorly/not-at-all documented modifications!